travis/upload_logs.sh

```diff
@@ -12,5 +12,9 @@ docker cp --follow-link $mesos_master_container:/var/log/mesos-master.INFO ./mes
 docker cp --follow-link $mesos_master_container:/var/log/mesos-master.WARNING ./mesos/master-logs/
 
 tarball=./dump.txz
-tar -cJf $tarball --transform="s|\./[^/]*/\.*|${TRAVIS_JOB_NUMBER}/|" ./scheduler/log ./travis/.minimesos ./mesos/master-logs
+tar -cJf $tarball --transform="s|\./[^/]*/\.*|${TRAVIS_JOB_NUMBER}/|" --warning=no-file-changed ./scheduler/log ./travis/.minimesos ./mesos/master-logs || exitcode=$?
+if [ "$exitcode" != "" ] && [ "$exitcode" != "1" ] && [ "$exitcode" != "0" ]; then
```

Comment:
Let's just check for `[ "$exitcode" == 2 ]` here. We can add a comment saying that GNU tar always exits with 0, 1 or 2 (https://www.gnu.org/software/tar/manual/html_section/tar_19.html).

Reply:
Done